src/main/kotlin/com/tally/kiosk/shop/persistence/ShopJpaRepository.kt

```
@@ -0,0 +1,7 @@
+package com.tally.kiosk.shop.persistence
+
+import com.tally.kiosk.shop.entity.Shop
+import org.springframework.data.jpa.repository.JpaRepository
+
+interface ShopJpaRepository : JpaRepository<Shop, Long> {
+}
```

detekt (1.23.7)

[warning] The class or object ShopJpaRepository is empty.

(detekt.empty-blocks.EmptyClassBlock)